frontend/src/components/RFI/RFILanding.vue

```diff
@@ -141,32 +141,14 @@
                 </v-col>
 
                 <v-col class="col-md-3 col-12">
-                  <v-menu
-                    v-model="calendarMenu[index]"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    min-width="auto"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field
-                        v-model="expense.date"
-                        :disabled="isReadOnly"
-                        :rules="rules.required"
-                        variant="outlined"
-                        label="Date of Expense (YYYY-MM-DD)"
-                        readonly
-                        v-bind="props"
-                      />
-                    </template>
-                    <v-date-picker
-                      v-model="expense.date"
-                      :disabled="isReadOnly"
-                      clearable
-                      @input="calendarMenu[index] = false"
-                    />
-                  </v-menu>
+                  <AppDateInput
+                    v-model="expense.date"
+                    :rules="rules.required"
+                    :disabled="isReadOnly"
+                    :hide-details="isReadOnly"
+                    label="Date of Expense (YYYY-MM-DD)"
```

Comment:
I think you update other labels from "Date of Expense (YYYY-MM-DD)" or "Date of Change (YYYY-MM-DD)" or "Date (YYYY-MM-DD)" to "Date". Do we have requirements to make that change? If yes, should we update this label as well?